Casks/blank-screensaver.rb

```diff
@@ -1,26 +1,32 @@
 cask 'blank-screensaver' do
-  version '2.1.0'
-  sha256 '068bafd2707cc9de3cdeedfc8f0d0637b2dbd86c0cb5acb1ff0fec595dd8211d'
+  version '2.1.1'
+  sha256 'fa35c8bd75f09a01abc27cf723bf56b3844ace6b5f911592e75bd9040861edff'
 
   url "https://github.com/theseal/macos-blank-screensaver/archive/#{version}.tar.gz"
   name 'Blank'
-
+  desc 'A blank (black) screensaver for MacOS/OS X.'
   homepage 'https://github.com/theseal/macos-blank-screensaver'
 
-  if MacOS.version == :mojave
+  if MacOS.version == :mojave || MacOS.version == :big_sur
     screen_saver "macos-blank-screensaver-#{version}/Blank.saver"
   else
     screen_saver "macos-blank-screensaver-#{version}/Blank.qtz"
   end
 
+  preflight do
+    File.delete('~/Library/Screen Savers/Blank.qtz') if File.exist?('~/Library/Screen Savers/Blank.qtz')
+    FileUtils.remove_dir('~/Library/Screen Savers/Blank.saver') if File.directory?('~/Library/Screen Savers/Blank.saver')
+  end
+
+  postflight do
+    if MacOS.version == :big_sur
+      system_command "xattr",
+        args: ["-d", "com.apple.quarantine", "#{ENV["HOME"]}/Library/Screen Savers/Blank.saver"]
```

Comment:
This will call `xattr -d com.apple.quarantine '~/Library/Screen Savers/Blank.saver` on Big Sur, which will remove it from quarantine and make it available to run. (At least it does for me on my machine.)